rules/TypeDeclaration/TypeAnalyzer/AlwaysStrictScalarExprAnalyzer.php

```diff
@@ -89,7 +88,7 @@ private function resolveCastType(Cast $cast): ?Type
 
     private function isScalarType(Type $type): bool
```

Comment:
you might try `$type->isScalar()->yes()`

Reply:
I will try 👍

Reply:
It seems got error on StringType:

```
There was 1 error:

1) Rector\Tests\TypeDeclaration\Rector\ClassMethod\ReturnTypeFromStrictScalarReturnExprRector\ReturnTypeFromStrictScalarReturnExprRectorTest::test with data set #7
Error: Call to undefined method PHPStan\Type\StringType::isScalarType()
```

Reply:
oh, it `isScalar()` instead of `isScalarType()`, I will create new PR.

Reply:
https://github.com/rectorphp/rector-src/pull/4920